MoltenVK/MoltenVK/Commands/MVKCmdPipeline.mm

```diff
@@ -193,10 +193,14 @@
 													   uint32_t firstSet,
 													   uint32_t setCount,
 													   const VkDescriptorSet* pDescriptorSets) {
+	if (_pipelineLayout) { _pipelineLayout->release(); }
```

Comment:
Meh. Sorry...I should have thought of this on the first review pass. 

We probably need to init all `_pipelineLayout` member vars in `MVKCmdPipeline.h` to `nullptr` to guard against this line possibly crashing on first invocation if `_pipelineLayout` contains memory garbage?

Reply:
Done.